pkgs/applications/virtualization/conmon/default.nix

```diff
@@ -0,0 +1,39 @@
+{ flavor ? ""
```

Comment:
Why are `flavor` and `ldflags` necessary? In particular, `flavor` is used to suffix the binary name, but unless you want to do something like "build 2 copies of `conmon` and put them both in `PATH` through `nix-env` without a conflict" -- which seems like a weird use case, IMO -- I don't see why you would want this. Generally if you want to change stuff like this you'd just do an `override`. I may not be understanding the use case here.

Reply:
The idea was to have multiple builds (like static ones) and flavour them (suffix them) with something like `-x86_64-static-glibc`. I think we can simply override and get rid of the flavor vars. I will adapt it.